src/components/gamePage/GamePage.tsx

```diff
@@ -0,0 +1,35 @@
+import { avatarsArray } from '../../store/basicMedia';
+import { useRoomState } from '../../store/roomStore';
+import { Players } from '../basicComponents/playersWidget';
+import style from './GamePage.module.css';
+
+export const GamePage = () => {
+  const players = useRoomState((state) => state.players);
+
+  return (
+    <div className={style.startPageWrapper}>
+      <Players />
+      <div className={style.tableWrapper}>
+        <div className={style.startTable}>
+          <div className={style.players}>
+            {players.map((el, index) => {
+              if (index !== 0) {

```

Comment:
Index is not what you want to decide if your client is at the bottom. You get your client's id from "get-me" response on connection to server. Use it to detect your player object from roomState.